### PR TITLE
Add procedure listing, plus a button for creating procedures

### LINKED ISSUE #110: Create interface to list all procedures
Something similar to Google Drive would be good for the first page that user sees when they log in

### PR DESCRIPTION
**Pull request by ConnorCimowsky**
Resolves #110 :sparkles:

### CODE REVIEW

File: src-frontend/app/controllers/procedures/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+    actions: {
+        createProcedure: function() {
+            var procedureController = this;
+            var procedure = this.store.createRecord('procedure', {
+                title: 'Surgery Follow-Up',
+                author: 'Partners for Care',
+                owner: 1
```
Comment (Trinovantes) [inline src-frontend/app/controllers/procedures/index.js]: add a TODO here for a separate github issue

**Comment by Trinovantes** [comment]
+1

File: src-frontend/tests/unit/components/procedure-card-test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { moduleForComponent, test } from 'ember-qunit';
```
Comment (Trinovantes) [inline src-frontend/tests/unit/components/procedure-card-test.js]: 4 spaces

Comment (Ommy) [inline src-frontend/tests/unit/components/procedure-card-test.js]: Having this as a separate issue to deal with all the tests generated. Can we configure ember generators to make it 4 spaces?

**Comment by Ommy** [comment]
lgtm, +1

File: src-frontend/tests/unit/components/procedure-card-test.js (hunk 2 above)
Comment (ConnorCimowsky) [inline src-frontend/tests/unit/components/procedure-card-test.js]: Probably. I'll look into it as part of the other issue.

**Pull request merged**